### PR TITLE
[#2754] improvement(web): list metalakes sort by created time as default

### PR DESCRIPTION
**Pull request by ibaiway**
### What changes were proposed in this pull request?

When listing metalakes they get sorted by their createdTime value

### Why are the changes needed?

Its requested in the issue
Fix: #2754  (issue)

### Does this PR introduce _any_ user-facing change?

metalakes will be shown in ASC order

### How was this patch tested?

N/A

### CODE REVIEW

**Review by ch3yne — APPROVED** [review]
I left a comment.

File: web/src/lib/store/metalakes/index.js [hunk 1]
```diff
@@ -38,6 +38,8 @@ export const fetchMetalakes = createAsyncThunk('appMetalakes/fetchMetalakes', as
 
   const { metalakes } = res
 
+  metalakes.sort((a, b) => new Date(a.audit.createtime) - new Date(b.audit.createtime))
```
Comment (ch3yne) [inline web/src/lib/store/metalakes/index.js:41]:
```suggestion
  metalakes.sort((a, b) => new Date(b.audit.createTime) - new Date(a.audit.createTime))
```
I think we should use descending order by time, and the field name should be `createTime`.

**Review by ch3yne — CHANGES_REQUESTED** [review]
`metalakes.sort((a, b) => new Date(b.audit.createTime) - new Date(a.audit.createTime))`

**Comment by ibaiway** [comment]
@ch3yne thanks for looking into this.
I agree that desc would be better so have updated the code accordingly.

**Review by ch3yne — APPROVED** [review]
Its good to me, thanks for your contribution! @ibaiway

**Pull request merged**